pkg/core/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -319,6 +319,24 @@ one = 'Kafka instance "{{.Name}}" already exists'
 [kafka.create.error.oneinstance]
 one = 'unable to create new Kafka instance. Only one instance is allowed with an eval account'
 
+[kafka.create.region.error.regionNotSupported]
+one = '''Selected region does not support the instance types that you can create. Region {{.Region}}
+Your instance types {{.MyTypes}}. 
+Region instance types {{.CloudTypes}}  
+'''
+
+[kafka.create.region.error.invalidRegion]
+one = '''
+Invalid or not enabled cloud provider region: "{{.Region}}"
+Valid region ids: {{.Regions}}
+'''
+
+[kafka.create.region.error.invalidProvider]
```

Comment:
![Screenshot 2022-01-19 at 16 15 03](https://user-images.githubusercontent.com/11743717/150170793-a4d54868-bba4-4fad-8562-1714de5298af.png)

I'm seeing this error when I pass aws (a valid provider) and an invalid region. I think it should be the other way around.